deployments/azure/templates/bicep/all-in-one/all-in-one-with-alb.bicep

```diff
@@ -0,0 +1,328 @@
+targetScope = 'subscription'
```

Comment:
Very nice!

I just tested this all-in-one-with-alb arm template and verified it deploys successfully (using dev workspaces docker image with arm changes - `aliciadnguyen/regulus-workspaces:169a0d5`):
<img width="878" alt="Screenshot 2024-04-15 at 11 54 50 AM" src="https://github.com/Teradata/ai-unlimited/assets/166751279/8cf2bad3-029f-4663-ae45-366433420a81">

Reply:
I tried passing in `ApplicationgateWayCertificate`, but I am running into this:
<img width="1547" alt="Screenshot 2024-04-15 at 12 15 58 PM" src="https://github.com/Teradata/ai-unlimited/assets/166751279/91043665-16a5-4891-9c8b-173ee4dd6b5c">

<img width="582" alt="Screenshot 2024-04-15 at 12 16 18 PM" src="https://github.com/Teradata/ai-unlimited/assets/166751279/5416d20d-77a0-48e2-8789-9f9bdc969c81">

I used this command to create the SSL certs: `openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 365` and then get the base64 of the cert file. 

I believe I am not doing it correctly. Would you have the command for me to create the cert? Thank you.